en/datastore.rst

```diff
@@ -25,6 +25,17 @@ The Datastore has three APIs:
 
 Anyone can access the Datastore – just build a query and data will be returned.
 
+Alpha Status
+------------
+
+Please be aware that the Datastore is in alpha stages. This means that it may exhibit unexpected behaviour or return unexpected results. This product is intended for user testing and feedback only and therefore should not be considered a stable product. The Datastore is therefore not feature complete, and further work on this product will resume in the future.
```

Comment:
therefore, therefore, therefore, thereform

Reply:
Suggestion:

> Please be aware that the Datastore is in alpha. `This means <https://en.wikipedia.org/w/index.php?title=Software_release_life_cycle&oldid=806157136#Alpha>`__ that it may be unstable and could cause crashes or data loss. It should be used for exploratory and investigatory purposes, and not as part of a production system. If IATI data is required via an API, the `IATI Registry <https://iatiregistry.org/>`__ should be used.

Also, under `Current known issues include:`:

> The Datastore is not feature-complete

Reply:
That's not an issue, that's just a progress update. But duly noted on the 'therefore's.